backend/sim/wallet/address.go

```diff
@@ -112,7 +114,16 @@ func (a *Address) MarshalBinary() ([]byte, error) {
 // Decode decodes an address from an io.Reader. Part of the
 // go-perun/pkg/io.Serializer interface.
 func (a *Address) Decode(r io.Reader) error {
-	data := make([]byte, 64)
+	var length int64
```

Comment:
`int64`? why not `uint64`? could we use something shorter here, like `uint16`?

Reply:
Updated to use uint16 in b6137b1cdfaa284311fdb26a48d38c0ce24f1508. However, this commit will not be updated. Because, the decoding part is moved to `pkg/io` when introducing BinaryUnmarshaler in the later commit.